pages/login.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Head from 'next/head'
+import LoginForm from '../components/LoginForm'
+import styles from '../styles/Login.module.css'
+import router from 'next/router'
+
+const login = () => { 
+  const loginRequest = (formData: React.FormEvent<HTMLInputElement>) => {
+    console.log("login", formData)

```

Comment:
Customary to remove console.log() or anything that prints to the console when you are merging to main. Always imagine that what goes here will be seen by customers/clients.